### ISSUE TITLE
"Source folder does not exist" when backing up from a remote system via ssh

### ISSUE THREAD

**Issue by chri2** [basnijholt/rsync-time-machine.py#36]
I downloaded `rsync-time-machine.py` and tried the following:

```
someuser@pureos:~/tmp/backup$ ./rsync_time_machine.py 'root@192.168.180.2:/data' target
rsync-time-machine.py: [ERROR] Source folder '/data' does not exist - aborting.
```

But I'm quite sure the source folder does exist:

```
someuser@pureos:~/tmp/backup$ ssh 'root@192.168.180.2' ls -ld /data
drwxr-xr-x 5 root root 4096  3. Sep 19:56 /data
someuser@pureos:~/tmp/backup$ ssh 'root@192.168.180.2' ls -l /data
insgesamt 12
drwxr-xr-x  6 root root 4096 16. Sep 2022  home
drwxr-xr-x  5 root root 4096 12. Sep 16:25 opt
drwxr-xr-x 14 root root 4096 30. Aug 2022  var
```

I then thought that the script maybe looks for the folder locally on my system instead of checking the existence on the remote system and tried the following:

```
someuser@pureos:~/tmp/backup$ ./rsync_time_machine.py 'root@192.168.180.2:/data' target
rsync-time-machine.py: Safety check failed - the destination does not appear to be a backup folder or drive (marker file not found).
rsync-time-machine.py: If it is indeed a backup folder, you may add the marker file by running the following command:
rsync-time-machine.py: ssh -p 22 root@192.168.180.2 'mkdir -p -- "target" ; touch "target/backup.marker"'
someuser@pureos:~/tmp/backup$ ls target/backup.marker
target/backup.marker
```

Seems to indicate that the source folder when backing up from a ssh source seems to be checked locally.

**Comment by chri2** [comment]
Followed up by https://github.com/basnijholt/rsync-time-machine.py/issues/37

**Comment by basnijholt** [comment]
Took a while but this is fixed in v1.3.1 via #46
